sormas-ui/src/main/java/de/symeda/sormas/ui/caze/importer/CaseImporter.java

```diff
@@ -445,7 +447,12 @@ private void insertColumnEntryIntoData(CaseDataDto caze, PersonDto person, Strin
 						Pair<DistrictReferenceDto, CommunityReferenceDto> infrastructureData =
 							ImporterPersonHelper.getDistrictAndCommunityBasedOnFacility(pd.getName(), caze, person, currentElement);
 						List<FacilityReferenceDto> facility = FacadeProvider.getFacilityFacade()
-							.getByName(entry, infrastructureData.getElement0(), infrastructureData.getElement1(), false);
```

Comment:
I just notice that there is a second problem here. The imported facility could also be the occupation facility of the person. This has to be handled separately.

Reply:
The facilityType is already before the facility and of course the column is also in the template, I checked that. Using Hospital as fallback is not a good idea, first of all it would be quite magical behaviour and secondly it will lead to problems in the UI.